test/states/test.ContentPlayback.js

```diff
@@ -0,0 +1,94 @@
+import QUnit from 'qunit';
+import {ContentPlayback} from '../../src/states.js';
+
+/*
+ * These tests are intended to be isolated unit tests for one state with all
+ * other modules mocked.
+ */
+QUnit.module('ContentPlayback', {
+  beforeEach: function() {
+    this.events = [];
+    this.played = false;
+
+    this.player = {
+      paused: () => false,
+      play: () => {
+        this.played = true;
+      },
+      trigger: (event) => {
+        this.events.push(event);
+      },
+      ads: {
+        debug: () => {}
+      }
+    };
+
+    this.contentPlayback = new ContentPlayback(this.player);
+    this.contentPlayback.transitionTo = (newState) => {
+      this.transitionTo = newState.name;
+    };
+  }
+});
+
+QUnit.test('only plays on init on correct conditions', function(assert) {
+  this.player.paused = () => false;
+  this.player.ads._cancelledPlay = false;
+  this.player.ads._pausedOnContentupdate = false;
+  this.contentPlayback.init(this.player);
+  assert.equal(this.played, false);
```

Comment:
this doesn't seem right since `paused() -> false`

Reply:
maybe `played` should be `playTriggered`

Reply:
changing it